packages/client/wallets/aa/package.json

```
@@ -40,5 +40,5 @@
     "sideEffects": false,
     "type": "module",
     "types": "./dist/index.d.ts",
-    "version": "0.1.2"
-}
\ No newline at end of file
+    "version": "0.1.1"
```

let's make sure this stays 0.1.2
```suggestion
    "version": "0.1.2"
```